src/Compiler/TypedTree/TypedTreeOps.fsi

```diff
@@ -2676,3 +2676,6 @@ type TraitConstraintInfo with
 
     /// Get the key associated with the member constraint.
     member GetWitnessInfo: unit -> TraitWitnessInfo
+
+val (|EmptyModuleOrNamespaces|_|):
```

Comment:
Can we get a docstring for this? Since it considers a module/namespace with a `TMDefDo` as empty, which it is from point of view of a signature, but maybe not in some other context.